### DISCUSSION TITLE
React-calendar language - always in Icelandic.

### DISCUSSION THREAD

**Discussion by EwaDwornik** [wojtekmaj/react-calendar#770] [Q&A] [answered]
Edit: problem solved by adding locale="is-IS".

Thank you for making react-calendar, it is really useful. I would love to set up the language to Icelandic, always, no matter what a user he's in their locale. Is it possible?
Best regards :)

**Comment by wojtekmaj** [comment, marked as answer]
Just to mark question as answered - add `locale="is-IS"` prop :)
